deployments/jwebbinar/image/environments/setup-notebooks

```diff
@@ -1,6 +1,4 @@
-#! /bin/bash -eu
-
-# This is a standard entry-point used to set up notebooks for this
+s is a standard entry-point used to set up notebooks for this
```

Comment:
You're probably already aware but some of the notebooks defined as tests below didn't pass,  including one failure due to a missing package (stcal) which might be easy to fix.